README.md

```diff
@@ -2,195 +2,240 @@
-    Language:        TextProto
-    BasedOnStyle:    google
-ReflowComments:  true
-SortIncludes:    false
-SortUsingDeclarations: true
+
 SpaceAfterCStyleCast: true
+SpaceAfterLogicalNot: false
 SpaceAfterTemplateKeyword: true
+SpaceAroundPointerQualifiers: Default
 SpaceBeforeAssignmentOperators: true
+SpaceBeforeCaseColon: false
+SpaceBeforeCpp11BracedList: false
+SpaceBeforeCtorInitializerColon: true
+SpaceBeforeInheritanceColon: true
 SpaceBeforeParens: ControlStatements
+SpaceBeforeRangeBasedForLoopColon: true
+SpaceBeforeSquareBrackets: false
+SpaceInEmptyBlock: false
 SpaceInEmptyParentheses: false
 SpacesBeforeTrailingComments: 1
-SpacesInAngles:  false
-SpacesInContainerLiterals: true
+SpacesInAngles:  Never
+SpacesInConditionalStatement: false
+SpacesInContainerLiterals: false
 SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
-Standard:        Cpp11
-TabWidth:        8
-UseTab:          Never
-...
+SpacesInLineCommentPrefix:
+   Minimum:         0
+   Maximum:         -1
+
+BitFieldColonSpacing: Both
+Cpp11BracedListStyle: true
+
+FixNamespaceComments: true
+ShortNamespaceLines: 0
+
+SortIncludes:    Never
+SortUsingDeclarations: true
+
+PointerAlignment: Left
+ReferenceAlignment: Pointer
+
+EmptyLineAfterAccessModifier: Never
+EmptyLineBeforeAccessModifier: LogicalBlock
+KeepEmptyLinesAtTheStartOfBlocks: false
+MaxEmptyLinesToKeep: 1
+#SeparateDefinitionBlocks: Always
+
+AttributeMacros:
+  - __capability
+  - __extension__
+  - __const__
+  - __volatile__
+  - __signed__
+  - __inline__
+  - BOOST_RESTRICT
+  - BOOST_FORCEINLINE
+  - BOOST_NOINLINE
+  - BOOST_NORETURN
+  - BOOST_ATTRIBUTE_UNUSED
+  - BOOST_ATTRIBUTE_NODISCARD
+  - BOOST_ATTRIBUTE_NO_UNIQUE_ADDRESS
+  - BOOST_MAY_ALIAS
+  - BOOST_NO_MAY_ALIAS
+  - BOOST_SYMBOL_VISIBLE
+  - BOOST_SYMBOL_EXPORT
+  - BOOST_SYMBOL_IMPORT
+  - BOOST_INLINE_VARIABLE
+  - BOOST_INLINE_CONSTEXPR
+  - BOOST_NOEXCEPT_OR_NOTHROW
+  - BOOST_FINAL
+  - BOOST_OVERRIDE
+ForEachMacros:
+   - foreach
+   - BOOST_FOREACH
+   - BOOST_REVERSE_FOREACH
+IfMacros:
+   - CATCH_CHECKED_IF
+   - CATCH_CHECKED_ELSE
+   - CHECKED_IF
+   - CHECKED_ELSE
+   - BOOST_IF_CONSTEXPR
+StatementMacros:
+   - BOOST_PRAGMA_MESSAGE
+   - BOOST_HEADER_DEPRECATED
+WhitespaceSensitiveMacros:
+   - STRINGIZE
+   - PP_STRINGIZE
+   - BOOST_STRINGIZE
+   - BOOST_PP_STRINGIZE
+   - BOOST_PP_WSTRINGIZE
+   - CMAKE_STRINGIFY
+   - CMAKE_TOSTRING
+
+PenaltyBreakAssignment: 2
+PenaltyBreakBeforeFirstCallParameter: 0
+PenaltyBreakComment: 300
+PenaltyBreakFirstLessLess: 120
+#PenaltyBreakOpenParenthesis: 0
+PenaltyBreakString: 1000
+PenaltyBreakTemplateDeclaration: 10
+PenaltyExcessCharacter: 1000000
+PenaltyIndentedWhitespace: 0
+PenaltyReturnTypeOnItsOwnLine: 200
 
+CommentPragmas:  '^ (IWYU pragma:|(NOLINT(NEXTLINE|BEGIN|END)?)|clang-format)'
+CompactNamespaces: false
+IncludeBlocks:   Preserve
+InsertTrailingCommas: None
+ReflowComments:  true
+#RequiresClausePosition: OwnLine
 ```
```

Comment:
Why do we have the contents of .clang-format reproduced in the README?

Reply:
That is just how it was originally. Can remove if people feel it is unnecessary/wrong.

Reply:
yeah, let's just link to the real file instead of duplicating the content.  no lag / drift.

Reply:
Sure thing.

Reply:
Removed it.

I left off the link to the actual file since it's right there.